odl/tomo/geometry/conebeam.py

```diff
@@ -16,15 +16,16 @@
 
 import numpy as np
 
+from odl.discr import uniform_partition, nonuniform_partition
 from odl.tomo.geometry.detector import Flat1dDetector, Flat2dDetector
 from odl.tomo.geometry.geometry import (
     DivergentBeamGeometry, AxisOrientedGeometry)
 from odl.tomo.util.utility import euler_matrix, transform_system
 from odl.util import signature_string, indent_rows
 
 
-__all__ = ('FanFlatGeometry',
-           'CircularConeFlatGeometry', 'HelicalConeFlatGeometry',)
+__all__ = ('FanFlatGeometry', 'ConeFlatGeometry',
+           'cone_beam_geometry')
```

Comment:
Maybe we should add a "helical_geometry" that automatically selects pitch, pitch_offset etc to fit a volume.

Perhaps not in this PR though.